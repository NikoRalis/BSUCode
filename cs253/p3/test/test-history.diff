5,9c5,6
< [0] pwd
< 0 ls
< 1 cd
< 2 pwd
< 3 history
---
> /home
> [0] history
14d10
< /home/marissa
18,28c14,15
< [0] pwd
< 0 pwd
< 1 history
< 2 pwd
< 3 history
< 4 touch
< 5 cat
< 6 cd
< 7 clear
< 8 ls
< 9 time
---
> /home
> [0] history
