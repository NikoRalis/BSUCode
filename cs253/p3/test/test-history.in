ls -l -a
cd /home
pwd
history
touch hello.txt
cat hello.txt
cd
clear
ls
time
pwd
history
exit
