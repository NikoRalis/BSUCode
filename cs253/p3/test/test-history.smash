[0] ls
[1] -l
[2] -a
/home
/home
[0] history
[0] touch
[1] hello.txt
[0] cat
[1] hello.txt
[0] clear
[0] ls
[0] time
/home
[0] history
