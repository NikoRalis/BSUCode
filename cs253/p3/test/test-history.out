[0] ls
[1] -l
[2] -a
/home
[0] pwd
0 ls
1 cd
2 pwd
3 history
[0] touch
[1] hello.txt
[0] cat
[1] hello.txt
/home/marissa
[0] clear
[0] ls
[0] time
[0] pwd
0 pwd
1 history
2 pwd
3 history
4 touch
5 cat
6 cd
7 clear
8 ls
9 time
